textalyze.js

```diff
@@ -16,18 +16,20 @@ const itemCounts = array => array.reduce((counts, item) => !counts.has(item)
   : counts.set(item, counts.get(item) + 1)
   , new Map());
 
+const stringToLetterArray = str => str.split('');
```

Comment:
Great.  For practice, write a few tests for this function in the same style as the tests in `__tests__`.